Mixpanel/MPNotificationViewController.m

```diff
@@ -440,6 +449,10 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
     if (self = [super initWithCoder:aDecoder]) {
         _maskLayer = [GradientMaskLayer layer];
         [self.layer setMask:_maskLayer];
+        [_maskLayer setColors:@[(id)[UIColor blackColor].CGColor,(id) [UIColor blackColor].CGColor,(id) [UIColor clearColor].CGColor,(id) [UIColor clearColor].CGColor]];
```

Comment:
`setColors` is expecting an array of `CGColorRef` and `CGColor` is an struct :( I know, it looks very lame but wanted to make clear we are passing the right struct. I'll pass `UIcolor` so it does not as ugly